guides/micronaut-jms-aws-sqs/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Connect a Micronaut JMS Application to an AWS SQS Queue",
+  "intro": "Learn how to connect JMS Application to an AWS SQS Queue",
+  "authors": ["Slavko Bodvanski"],
+  "tags": ["jms", "messaging", "aws"],
+  "categories": ["Messaging"],
+  "maximumJavaVersion": 16,
```

Comment:
why fix maximumJavaVersion?

Reply:
Until we add a new module to micronaut-jms to support Java AWS SDK v2's, it seems that we are limited to Java version below 17:
https://github.com/aws/aws-sdk-java#maintenance-and-support-for-java-versions